extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -21,58 +21,58 @@ class CakeRelatedContentService {
 	 * @param $ignore
 	 * @return RecirculationContent[]
 	 */
-	public function getContentRelatedTo($title, $universeName=null, $limit=5, $ignore=null) {
+	public function getContentRelatedTo( $title, $universeName = null, $limit = 5, $ignore = null ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)